Manual.md

```diff
@@ -113,56 +70,317 @@ Each container directory must be on a btrfs filesystem, and is created as a btrf
+
+```
+export PGARCHIVE=/path/to/container
+```
+
+For high-load DBs at [emarsys.com] the following [btrfs mount options] work best. You do need `user_subvol_rm_allowed` for snapshot expiry to work, the rest merely improve performance for us.
+
+```
+user_subvol_rm_allowed,noatime,nobarrier,enospc_debug,nodatacow,metadata_ratio=6
+```
+
+Now you need to use `pgarchive container init`, possibly with more configuration, to create container(s). Re-starting containers on server re-boots is left up to you. (TODO: add sysV and systemd service scripts to repository)
+
+### Security Considerations
+
+There are no internal security mechanisms at all. Note especially:
+
+* This is expected to be run on a trusted network. You could secure the upstream DB connections using SSL, but this is up to you.
+* By default backup containers can log into their upstream PostgreSQL instance using a super user, although you could reduce this to replication only after container initialization and replication slots have been created.
+* Access to containers and their data is purely host based. Data is not encrypted. Guard your pgarchive servers like the real thing.
+
+### Monitoring Hints
+
+Backup services need monitoring. How exactly this is done is out of scope of pgarchive and determined by local policy, but here is what we found useful at [emarsys.com], for each container:
+
+* The wal_archive directory should have frequent activity (mtime). If your upstream DB has long idle periods setting [archive_timeout] will put a limit on this (like 1 hour). Typically the wal_archive subsystem is not running if this alert is raised, or otherwise cannot write to the wal_archive directory.
+
+* The snapshots directory should have frequent activity too, depending on the cron job for snapshot creation. Note snapshot creation only succeeds if the standby process is active, which means running and having a recent restartpoint, so this typically flags standby process problems.
+
+* You absolutely need to monitor the size of your upstream DB's $PGDATA/pg_xlog directory. But you already do this, don't you? If offline pgarchive containers cannot stream replication data, old WAL segments cannot be released and will fill up pg_xlog [replication slots]. You may eventually need to decide to drop the replication slot, which will break synchronization with your container.
+
+Of course you should monitor free disk space for your containers, and other common system parameters.It's probably less useful to check for the presence of [pg_receivexlog] and PostgreSQL standby processes, because it's hard to associate them with multiple containers easily, and this may miss certain problems if subsystems become stuck without dying fully.
+
+
+## Basic Operation
+
+`pgarchive` is a command line utility. It is intended to be used by the postgres user, and it contains a built in `help` command which outputs a summary of all (public) commands. See the Command Reference below.
```

Comment:
i'd move some information from here up into the overview introduction section, e.g. that pgarchive is actually "only" a shell script using / glueing together postgres standard components which then also make up the server / daemon parts of the archive system. 

this will be interesting for everybody who does not already understand the architecture -- especially since you introduce `pgarchive` as PG Archive _Server_.